nvme.c

```diff
@@ -7415,6 +7415,23 @@ static int copy_cmd(int argc, char **argv, struct command *cmd, struct plugin *p
 	return err;
 }
 
+static void io_cmd_show_error(struct nvme_dev *dev, int err)
+{
+	if (err > 0) {
+		nvme_show_status(err);
+		return;
+	}
+
+	nvme_show_init();
+
+	nvme_show_error("flush: %s", nvme_strerror(errno));
+
+	if (is_chardev(dev))
+		nvme_show_result("chardev handled so needed to use blkdev instead");
```

Comment:
Maybe add something like "chardev handled so needed to use blkdev instead, e.g. /dev/nvme0n1" ?

Reply:
Changed the message as suggested. Thank you.